docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,498 @@
+
+To show a pop-up with a message, use `lcResume`. Remove all the lines related to other events except `lcResume`.
+
+![JavaScript file for resume](media/js_for_resume.png)
+
+You need to use `addonapi`, which includes APIs for addons. To show the pop-up, you can use `addonapi.window.showMessageBox()`. For example:
+
+```
+const addonAPI = require('addonapi');
+
+resume(winId) {
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'Resume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+
+
+To verify whether the modified file works as expected, follow these steps:
+
+1. Using the changed file, build a signed package again.
+
+2. Before installing the newly built package, remove the installed addon.
+
+3. In Addon Settings App, to remove the installed addon, click **X** button (![addon settings x button](media/addon_settings_x_btn.png)).
+
+4. To install and run the newly built addon package, follow the [steps to run addon](#run-addon) again. 
+
+    The following pop-up apprears:
+
+    **Figure: Addon Pop-up on Resume Event in Basic UI**
+
+    ![Showing pop-up in Basic UI](media/hello_resume.png)
+
+
+<a name="load-html-from-addon"></a>
+### Load HTML file from addon
+
+In an addon, you can load your own HTML file. In this section, an example of an HTML file that shows `Hello, I'm an Addon` is explained.
+
+Ensure that all the files related to an addon are located under `shared\res` of your addon project. Therefore, the added HTML file must also be located in the same directory.
+
+To add a new HTML file for addon, unfold the BasicAddon in Project Explorer. Then, right-click on the shared/res > New > Files and enter the file name as `hello.html`.
+
+![Add a new file](media/new_file.png)
+
+![Generate the HTML file](media/generate_hello_html.png)
+
+Following is an example of `hello.html` HTML file:
+
+```
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, width=device-width">
+</head>
+<body>
+    <h1>Hello, I'm an Addon</h1>
+</body>
+</html>
+```
+
+The HTML file must be loaded in the addon. To do so, you can use `addonapi.window.loadURL()` when the resume event occurs. Following is the code for the `rusume()` event:
+
+```
+resume(winId) {
+    console.log('TmplDefault.resume');
+    addonAPI.window.loadURL(winId, 'file://' + __dirname + '/hello.html');
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'TcResume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+
+Including the changes above, build a new package and install it.
+When you run the Web application, you will see the following screen:
+
+![Resume with HTML and pop-up](media/resume_html_popup.png)
+
+### Return to main page of Web application
```

Comment:
@hyundukkim 
While reading, I am getting an idea that there are two options to return to the main page of the Web application:
1.	Resolving two issues in hello.html
2.	Using the Back key

If that is right and you agree, then can we represent it in a better way?
For example,
### Return to main page of Web application
When you implement to [load HTML file from addon](#load-html-from-addon), you will not be able to return to the main page of the Web application. This is because it loads `hello.html` in every resume event and does not know the URL of the main page of the Web application. To return to the main page, you can use one of the following ways:
1.	Resolving issues in addon events
2.	Using the Back key

#### Resolve issues in addon events
To return to the main page of the web application, resolve the following two issues:
    - Avoid showing the HTML page in every resume event
        The HTML page appears...
    - Load the URL of the main page of the Web application
         The HTML page appears...
#### Return to main page using back key
When you...

This is a sample format to represent the content, please verify and suggest appropriate headings also.
Please correct me if I am wrong.

@mijin85cho 
Please verify and share your valuable feedback.

Reply:
That seems to be okay if it has one correction.

**Avoid showing the HTML page in every resume event** must be common on both options.

Reply:
@mijin85cho , @Reni-90 ,

This is my preference based on @mijin85cho 's suggestion:

 ### Return to main page of Web application
 
 When you implement to [load HTML file from addon](#load-html-from-addon), you will not be able to return to the main page of the Web application. This is because it loads `hello.html` in every resume event and does not know the URL of the main page of the Web application. To return to the main page, first avoid showing the HTML page in every resume event.
 
 The HTML page appears... _(The content of "Avoid showing the HTML page in every resume event")_
 
 With this modification, you can use one of the following ways to load the main page:
 
     1. Using occurrence of addon event
 
     2. Using the Back key
 
 
 #### Using occurrence of addon event
 
 The HTML page appears...
 #### Return to main page using back key
 
 When you...
 

@mijin85cho , @Reni-90 
Please verify it.

Reply:
I think @hyundukkim's suggestion is acceptable. @Reni-90 Would you please let us know your opinion about it?

Reply:
@Reni-90 I also agree with your suggestion.
@hyundukkim Would you please check the latest suggestion and let us know your opinion?

Reply:
@Reni-90 , @mijin85cho ,
Thank you for your additional suggestion.
I agree on them and have applied.